src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt

```diff
@@ -181,12 +179,6 @@ object ActiveServerList : Listenable {
     internal var serverList = ServerList(mc).apply { loadFile() }
 
     private val serverListPinger = MultiplayerServerListPinger()
-    private val serverPingerThreadPool: ThreadPoolExecutor = ScheduledThreadPoolExecutor(
-        10,
-        ThreadFactoryBuilder().setNameFormat("Server Pinger #%d")
-            .setDaemon(true)
-            .build()
-    )
```

Comment:
I should not have approed this PR. There is a reason why we use a scheduled thread pool, not ... spawn threads.

Reply:
or we just keep it as intended.

Reply:
it broke my whole server list